docs/Proposals.md

```diff
@@ -0,0 +1,30 @@
+# Proposals
+
+Although Windranger aims to provide a modular framework for DAO governance, we will adopt and experiment
+with a few simple approaches.
+
+## Process #1
+
+1. Propose on forum
+2. If forum responses appear positive, create snapshot governance proposal
```

Comment:
We do this onchain or offchain? It seems like there is informal forum voting, then official onchain voting, when proposal was created.

Reply:
There won't be any voting within the forum, it's a way to engage the community and to collect their sentiment.

Snapshot is mostly off-chain. It follows requirements such as minimum token balance to create a proposal. If the proposal does require on-chain execution, the txs to execute can optionally be encoded as part of the proposal, or it can be done manually. 

For a signalling proposal, the snapshot system can be used solely to execute an informal (mostly) off-chain vote without executing on-chain txs. It'll filter the community to those who actually hold BIT tokens, whereas forum participants may not hold any BITs.